WordPress/gutenberg
# Block Library: Add the Comments Pagination block

## Description

This block works as a container for other pagination blocks that render links to navigate through the list of comments. It basically works the same as the [Query Pagination block](packages/block-library/src/query-pagination), but for comments.

It has the same settings:
- changing layout and colors
- choosing different styles for next/prev arrows. 

Will close #35007

## How has this been tested?

1. Modify the Single Post template, adding a Comments Query Loop block
2. Add inside the Comments Pagination block
3. Check that the layout and color settings work (tested only in the editor)
4. Check that the arrow setting works as expected (tested only in the editor)

## Types of changes
- New feature

## Checklist:
